src/rules/checkLinesAlignment.js

```diff
@@ -0,0 +1,203 @@
+import {
+  set,
+} from 'lodash';
+import iterateJsdoc from '../iterateJsdoc';
+
+/**
+ * Create a sequence of chars with a specific size.
+ *
+ * @param {string} char   The char.
+ * @param {int}    length The length.
+ */
+const createSequence = (char, length) => {
+  return new Array(length + 1).join(char);
+};
+
+/**
+ * Aux method until we consider the dev envs support `String.prototype.matchAll` (Node 12+).
+ *
```

Comment:
Though the rule unfortunately does not yet support jsdoc-style `@todo`, Unicorn has an excellent rule we are using, [`expiring-todo-comments`](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/expiring-todo-comments.md) which can turn the to-do comment into an error as soon as a condition is matched--in this case when `package.json` starts pointing to Node 12 as a minimum.

You can use this by putting the following above this block instead.

```js
// todo[engine:node@>=12]: Replace with `String.prototype.matchAll`.
```

This has the advantage of ensuring that we will be reminded to make the change as soon as we can.

Reply:
Pretty interesting! TIL =)
Comment added here: https://github.com/gajus/eslint-plugin-jsdoc/pull/636/commits/0be5a14e68026839e270d6ac6fd8df60fa83b274